src/vm/hints/hint_utils.rs

```diff
@@ -28,6 +28,29 @@ pub fn get_int_from_scope(vm: &mut VirtualMachine, name: &str) -> Option<BigInt>
     val
 }
 
+pub fn get_u64_from_scope(vm: &mut VirtualMachine, name: &str) -> Result<u64, VirtualMachineError> {
+    let mut val: Result<u64, VirtualMachineError> = Err(VirtualMachineError::ScopeError);
+    if let Some(variables) = vm.exec_scopes.get_local_variables() {
+        if let Some(PyValueType::U64(py_val)) = variables.get(name) {
+            val = Ok(*py_val);
+        }
+    }
+    val
+}
+
+pub fn get_u64_from_scope_ref<'a>(
+    vm: &'a mut VirtualMachine,
+    name: &'a str,
+) -> Result<&'a u64, VirtualMachineError> {
```

Comment:
Copying a `u64` is actually cheaper than returning a reference:
```suggestion
) -> Result<u64, VirtualMachineError> {
```

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/319/commits/6607a39a7d71d67b517356a9ca564887b6c04cfa